base/server/python/pki/server/cli/acme.py

```diff
@@ -607,7 +608,7 @@ def execute(self, argv):
         database_type = DATABASE_TYPES.get(database_class)
         print('  Database Type: %s' % database_type)
 
-        if database_type == 'ldap':
+        if database_type == 'ldap' or database_type == 'openldap':
```

Comment:
Should we split into three instead of two?

`ldap` -- generic base, `389ds` and `openldap`? Then if there are any overtly 389ds-specific things, we can do a "general" implementation that might work on LDAPMcLDAP's implementation, and then overwrite it with 389-ds specific one too? Comment kinda applies everywhere, not just here. 

i.e., make it explicit that `ldap` really means 389ds? Then making openldap seems more natural.

But up to you.

Reply:
Actually I was considering to rename `ldap` to `ds` after this PR, but it needs some discussions:
* `389ds` refers to the upstream project. Downstream it's `rhds`. So should we use `ds` instead? The class name will be`DSDatabase`.
* I'm not sure if we want to keep the generic `ldap` option since there's no product that can be used with it, but we can keep the `LDAPDatabase` as the base class.
* IPA will need to be updated as well.

Reply:
I've added the DS database. The doc has been updated as well.